packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {
+    OwnableUpgradeable
+} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { Semver } from "../universal/Semver.sol";
+
+/**
+ * @title SystemConfig
+ * @notice This contract is used to update L2 configuration via L1
+ */
+contract SystemConfig is OwnableUpgradeable, Semver {
+    uint256 public constant VERSION = 0;
+
+    uint256 public overhead;
```

Comment:
I think its simple to have these values be `uint256`s but I don't think that we really need them to be. The `GasPriceOracle` on L2 has them as `uint256` but making them smaller wouldn't break anything since its restricting the possible values, plus we haven't set them that large

Reply:
Changing them may affect contract storage layout - not a problem for this new contract, but there may be some other cases to consider, like updating op-geth to read the packed storage.